packages/xstate-wallet/src/workflows/application.ts

```diff
@@ -305,28 +311,30 @@ export const workflow = (
         outcome,
         appData,
         appDefinition,
-        fundingStrategy
+        fundingStrategy,
+        applicationSite
       } = context;
       const stateVars: StateVariables = {
         outcome,
         appData,
         turnNum: bigNumberify(0),
         isFinal: false
       };
-      const {channelId} = await store.createChannel(
+      const {channelId: targetChannelId} = await store.createChannel(
         participants,
         bigNumberify(challengeDuration),
         stateVars,
-        appDefinition
+        appDefinition,
+        applicationSite
       );
 
       // Create a open channel objective so we can coordinate with all participants
       await store.addObjective({
         type: 'OpenChannel',
-        data: {targetChannelId: channelId, fundingStrategy},
-        participants
+        data: {targetChannelId, fundingStrategy},
+        participants: [participants[1]]
```

Comment:
Why is this just one participant?

Reply:
Because if I am creating a channel, I don't want to also run the `OpenChannel` objective (which triggers a second application workflow)